src/Schema/Factories/FieldFactory.php

```diff
@@ -184,7 +184,7 @@ function () use ($argumentMap, $resolverWithMiddleware) {
                 $this->flushValidationErrorBuffer();
 
                 $argumentSet = $this->typedArgs->fromResolveInfo($this->args, $this->resolveInfo);
-                $spreadArguments = $argumentSet->spread();
+                $spreadArguments = $argumentSet->spread()->rename();
```

Comment:
```suggestion
                $spreadArguments = $argumentSet->spread();
                $renamedArguments = $spreadArguments->rename();
```

Another option would be to find a generic term for the result of chaining the operations, maybe `$modifiedArguments`?